WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -145,6 +189,209 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     }
 }
 
+
+#pragma mark - Private cell configuration methods
+
+- (NSString *)cellIdentifierForIndexPath:(NSIndexPath *)indexPath
+{
+    NSString *identifier = @"";
+    
+    StatsSection statsSection = [self statsSectionForTableViewSection:indexPath.section];
+    
+    switch (statsSection) {
+        case StatsSectionInsightsAllTime:
+            if (indexPath.row == 0) {
+                identifier = InsightsTableSectionHeaderCellIdentifier;
+            } else {
+                identifier = self.traitCollection.userInterfaceIdiom == UIUserInterfaceIdiomPad ? InsightsTableAllTimeDetailsiPadCellIdentifier : InsightsTableAllTimeDetailsCellIdentifier;
```

Comment:
I just realized that even though iOS 7 supports size classes it's ONLY through IB and not in code. This crashes on iOS 7.  Switching back to IS_IPAD for now.